### PR TITLE
pull_specific_messages function has been added to pull a specific num…

### PR DESCRIPTION
**Pull request by BrycenW**
…ber of messages.

### CODE REVIEW

File: inOut.py [hunk 1]
```diff
@@ -98,7 +98,24 @@ def pull_prev_message(current_id, token, group_id):
...
+def pull_specific_messages(current_id, token, group_id, number, after_bool):
+	if after_bool:
+		payload = {"limit":number, "after_id":current_id, "token":token}
+	else:
+		payload = {"limit":number, "before_id":current_id, "token":token}
+	try:
+		dirty_packet = requests.get("https://api.groupme.com/v3/groups/" +
+			group_id + "/messages", params=payload)
+	except requests.exceptions.ConnectionError:
+		time.sleep(.5)
```
Comment (CodaFi) [inline inOut.py]: Why does it need to sleep?

Comment (BrycenW) [inline inOut.py]: To prevent accidental dosing

**Pull request merged**